sources/packages/backend/libs/services/src/health-check/health.service.ts

```diff
@@ -0,0 +1,74 @@
+import { Injectable } from "@nestjs/common";
+import { RedisOptions, Transport } from "@nestjs/microservices";
+import {
+  HealthIndicator,
+  HealthIndicatorResult,
+  HealthCheckError,
+  MicroserviceHealthIndicator,
+} from "@nestjs/terminus";
+import { ConfigService } from "@sims/utilities/config";
+import { ZeebeHealthIndicator } from "./zeebe-health-indicator";
+
+@Injectable()
+export class HealthService extends HealthIndicator {
```

Comment:
For my knowledge, what is the benefit of extending `HealthIndicator`?

Reply:
HealthIndicator is a standard interface supplied by nest terminus for implementing health checks, also the getStatus method in it, returns a common object HealthIndicatorResult, which provides us with details like whether the health check is healthy and in case of failure, the reason for it, if properly set in the data field of the method.

Reply:
I can see it used for a custom indicator [here](https://docs.nestjs.com/recipes/terminus#custom-health-indicator) and my question was if we really need it.